xbmc/input/joysticks/JoystickMonitor.cpp

```diff
@@ -49,7 +51,9 @@ bool CJoystickMonitor::OnHatMotion(unsigned int hatIndex, HAT_STATE state)
 
 bool CJoystickMonitor::OnAxisMotion(unsigned int axisIndex, float position, int center, unsigned int range)
 {
-  if (position)
+  const bool bIsActive = (std::abs(position) >= AXIS_THRESHOLD);
```

Comment:
This calculation needs to change since the Peripheral API bump to 1.3.0. See https://github.com/xbmc/xbmc/blob/c80e266/xbmc/input/joysticks/generic/InputHandling.cpp#L69 for example.

This should be:

```c++
const float magnitude = std::abs(position - center) / range;

if (magnitude >= AXIS_THRESHOLD)
{
  ...
```